test/unit/runner.spec.js

```diff
@@ -416,7 +416,20 @@ describe('Runner', function() {
   });
 
   describe('allowUncaught', function() {
-    it('should allow unhandled errors to propagate through', function(done) {
+    var immediately;
+
+    before(function() {
+      immediately = Runner.immediately;
+      Runner.immediately = function(fn) {
```

Comment:
Using `before` or `beforeAll` for this suite of tests should make no difference nor affect in any way the one test we already had. I've opted for `before` because it executes less code, granted the gains are marginal. If you have a strong preference, I'm cool with switching to `beforeAll`.

Reply:
no, just wanted to know if there was an intent there, regarding "before all".

I believe you re: `immediately`; I just wonder if there's a better way to do this.  I'm not sure what that is yet.  we have to do this because the code under test is wonky, of course.

Reply:
BTW, the conflict in `lib/runner.js` might make this unnecessary.